README.md

```diff
@@ -95,8 +81,55 @@ When calling GET /am?prefix={complete-permission-here} your server should respon
 
 When an unauthorized request is made, a response with `{ "permission": {string}, "alias": {string} }` is expected.
 
+### The CI/CD pipeline
```

Comment:
Why is this nested under "About RBAC use cases"? It seems to me it should use h2 (`##`).

Reply:
👁 🦅